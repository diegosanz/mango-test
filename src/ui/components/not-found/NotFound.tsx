import React, { FC } from 'react';

const NotFound: FC = () => {
  return <p>La p√°gina solicitada no existe</p>;
};

export default NotFound;
