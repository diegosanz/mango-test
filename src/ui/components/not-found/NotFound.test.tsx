import React from 'react';
import { render } from '@testing-library/react';
import NotFound from './NotFound';
import TestRenderer from 'react-test-renderer';

test('renders NotFound', () => {
  render(<NotFound />);
});

test('snapshot match', () => {
  const tree = TestRenderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
