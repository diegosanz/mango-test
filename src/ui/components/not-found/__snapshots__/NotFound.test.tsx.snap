// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot match 1`] = `
<p>
  La p√°gina solicitada no existe
</p>
`;
